myhpi/static/scss/myHPI.scss

```diff
@@ -181,6 +181,15 @@ h1.side-panel-title::after {
     margin: -0.5rem -0.8rem;
 
     .search-result {
+        hgroup {
+          display: flex;
+          justify-content: space-between;

```

Comment:
So far we've tried to keep our scss to a minimum and instead use bootstrap's utility classes. You could use `d-flex justify-content-between`